src/gluonts/zebras/_freq.py

```diff
@@ -0,0 +1,174 @@
+# Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+
+from __future__ import annotations
+
+import re
+from dataclasses import dataclass
+from typing import Tuple
+
+import numpy as np
+
+from gluonts import maybe
+
+NpFreq = Tuple[str, int]
+
+
+def _canonical_freqstr(n: int, name: str) -> str:
+    """Canonical name of frequency.
+
+    >>> _canonical_freqstr("X")
```

Comment:
```suggestion
    >>> _canonical_freqstr(1, "X")
```

same below. Do these examples run as part of the docs build?